eradiate/scenes/biosphere/tests/test_biosphere_tree_mesh.py

```diff
@@ -0,0 +1,278 @@
+import os
+import tempfile
+
+import numpy as np
+import pytest
+
+from eradiate import unit_registry as ureg
+from eradiate.contexts import KernelDictContext
+from eradiate.scenes.biosphere._mesh_tree_element import MeshTreeElement
+from eradiate.scenes.biosphere._tree import MeshTree
```

Comment:
While you're at it, you can get an entry point check for free:
```suggestion
from eradiate.scenes.biosphere import MeshTreeElement
from eradiate.scenes.biosphere import MeshTree
```